salt/master.py

```diff
@@ -1276,6 +1276,14 @@ def __verify_load(self, load, verify_keys):
 
         return load
 
+    def _file_envs(self, *args):
+        '''
```

Comment:
Ok, so this should accept one keyword `load` and then add a pylint exception to ignore that `load` is not used.